openwisp_controller/config/api/urls.py

```diff
@@ -83,6 +83,21 @@ def get_api_urls(api_views):
                 api_download_views.download_device_config,
                 name='download_device_config',
             ),
+            path(
+                'controller/reversion/',
+                api_views.reversion_list,
+                name='reversion_list',
+            ),
+            path(
+                'controller/reversion/<str:pk>/',
+                api_views.reversion_detail,
+                name='reversion_detail',
+            ),
+            path(
+                'controller/reversion/<str:pk>/restore/',
```

Comment:
The URLs seem to be heavily focused on the `django-reversion` implementation rather than aligning with OpenWISP's approach.  

This was not the original intention of the issue. The goal is to introduce additional URLs for each resource (e.g., `Device`, `Template`, etc.) that supports reversion, allowing users to **list revisions, review revisions, and restore them**.  

For example, for `Template`:  
- `GET /controller/template/revisions/`  
- `GET /controller/template/revisions/{pk}/`  
- `POST /controller/template/revisions/{pk}/restore/`